tip-distributor/src/claim_mev_workflow.rs

```diff
@@ -55,13 +55,25 @@ pub fn claim_mev_tips(
     let mut transactions = Vec::new();
 
     runtime.block_on(async move {
-        let blockhash = rpc_client
-            .get_latest_blockhash()
-            .await
-            .expect("read blockhash");
-
+        let blockhash = rpc_client.get_latest_blockhash().await.expect("read blockhash");
```

Comment:
curious if reformat caused by your specific IDE settings

Reply:
not sure tbh, ran `cargo +nightly fmt` afterwards